frontend/www/src/js/modules/dashboard/tradition/propertyTableView.js

```diff
@@ -26,45 +27,98 @@ class PropertyTableView extends HTMLElement {
   }
 
   /** @type {TraditionMetaLabels} */
-  static #traditionMetadataLabels = {
+  static traditionMetadataLabels = {
     tradition: 'Tradition',
+    direction: 'Direction',
     owner: 'Owner',
     access: 'Access',
     language: 'Language',
-    witnesses: 'Witnesses'
+    witnesses: 'Witnesses',
+    name: 'Name'
   };
 
   /** @type {StemmaMetaLabels} */
   static #stemmaMetadataLabels = {
     stemma: 'Stemma'
   };
 
+  /** 
+   * @typedef {{
+   *    value: string, 
+   *    display:string
+   * }} SelectOption
+   * 
+   * @typedef {{
+   *    control: string,
+   *    size: integer,
+   *    required: boolean,
+   *    selectOptions: SelectOption[] 
+   * }} InputOptions
+   */
```

Comment:
From a typing point of view, we support three controls: `'text', 'checkbox' and 'dropdown'`. `selectOptions` is applicable only for `'dropdown'` controls. We can express this in JSDoc as follows:

(I also added an optional `label` property, since in some cases we define it, in some cases we do not.)

```suggestion
  /**
   * @typedef {{
   *   value: string;
   *   display: string;
   * }} SelectOption
   *
   * @typedef {{
   *   size: number;
   *   required: boolean;
   *   label?: string;
   * } & (
   *   | {
   *       control: 'dropdown';
   *       size: number;
   *       required: boolean;
   *       selectOptions: SelectOption[];
   *     }
   *   | {
   *       control: 'text' | 'checkbox';
   *       size: number;
   *       required: boolean;
   *     }
   * )} InputOptions
   */
```